lib/commands/command-plugin-list-all.bash

```diff
@@ -20,7 +20,7 @@ plugin_list_all_command() {
 
         printf "%s\\t%s\\n" "$index_plugin_name" "$installed_flag$source_url"
       done
-    ) | column -t -s $'\t'
+    ) | awk '{ printf("%-28s", $1); sub(/^[^*]/, " &", $2); $1=""; print $0 }'
```

Comment:
Why do these two awk commands differ? The `column` commands they replace are the same.

Reply:
This one:

1. prints the first column at 28 char width, left-aligned
2. Adds a space to column 2's without leading "*" for alignment

```
28-chars-wide                 next-column
28-chars-wide                *column-with-asterisk
```

The other one just prints the _first_ column at 28 char width, left-aligned